### PR TITLE
Feature: mark required, applicant column

### PR DESCRIPTION
**Pull request by yneves**
Introduced `DocumentInput` component to handle all input fields for documents. It takes care of adding required icon according to given flags. Also added the "Applicant" column.

### CODE REVIEW

**Comment by yneves** [comment]
Just to make it clear, this still doesn't handle PDF icon.

File: src/components/documents/DocumentInput.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+  };
+
+  shouldMarkRequired() {
+    return this.props.isSaved &&
+      !this.props.isDisabled &&
+      this.props.isRequired &&
+      !this.props.value;
+  }
+
+  render(field, isDisabled, isRequired) {
```
Comment (ntgn81) [inline src/components/documents/DocumentInput.js]: I didn't know render takes arguments, are they used this way?

**Comment by ntgn81** [comment]
Looks great! Doesn't work for DateInput yet, right? I have some custom logic to parse the date there. Like "1/1/05" will resolve into 01/01/2005

File: src/components/documents/DocumentInput.js (hunk 1 above)
Comment (yneves) [inline src/components/documents/DocumentInput.js]: Oh, forgot that when moving code around. Sorry.

**Comment by yneves** [comment]
I'll add it to `DateInput` and also handle "PDF" icon. I'll just make the border of the button red like the input, can't think of anything else.

**Comment by ntgn81** [comment]
ya it's tough, shall we merge this one for now?

**Comment by yneves** [comment]
Let me handle those two in this same PR. Almost done.

**Comment by ntgn81** [comment]
Okay

**Comment by yneves** [comment]
Fixed `th` and rebased.

**Comment by ntgn81** [comment]
Beautiful, merging

**Pull request merged**